src/components/Layout/dashboardMenuContainer/components/DashboardSubMenu.tsx

```diff
@@ -113,20 +115,23 @@ export default function DashboardSubMenu({ ownerName, description, role, selecte
                   </Typography>
                 </Grid>
                 <Grid item xs={10} mb={4}>
```

Comment:
When you have to render a text passed as input, you have not to think to localize it if you have not explicitly the necessity to concatenate it with text that require localization.